reactComponents/src/searchbar/tokenSearch/Chip.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, FC } from 'react';
+import styled from 'styled-components';
+import TokenSearchContext from '../Context/TokenSearch';
+
+import { Logo } from './Logo'
+import CheckedIcon from '../icons/checked'
+import UnCheckedIcon from '../icons/unchecked';
+
+const StyledChip = styled.div`
+    ${({ styles }) => `
+        > input {
+          display: none;
+        }
+
+        > input + label {
+          -webkit-transition: all 500ms ease;
+          transition: all 500ms ease;    
+          cursor: pointer;    
+          display: grid;
+          align-items: center;
+              
+          -moz-user-select: -moz-none;
+          -webkit-user-select: none;
+          -ms-user-select: none;
+          user-select: none;
+    
+          font-size: ${styles?.fontSize || "8px"};  
+          font-weight: ${styles?.fontWeight || "500"};  
+          border-radius: ${styles?.borderRadius || "4px"};  
+          background-color: ${styles?.backgroundColor || "#232B35"};  
```

Comment:
Should the hardcoded values derive from Tailwind?

Reply:
They are override style without Tailwind in the component.